scst-store/amr/graphql-query.hbs.md

```diff
@@ -0,0 +1,262 @@
+# AMR GraphQL Querying
+
+This page contains information on how to connect to the GraphQL playground, and how to query each supported data model.
+See [AMR Data Models](data-model-and-concepts.hbs.md) for more information on the AMR data models.
+
+## <a id='connecting-to-graphql'></a> Connecting to GraphQL Playground
+
+The recommended configuration is to enable ingress. The Supply Chain Security Tools for Tanzu – Store and Artifact Metadata Repository (AMR) packages share the same ingress configuration. 
+As such setting `ingress_domain` for the SCST - Store will apply the same value for the AMR.
+To connect to the AMR GraphQL playground when ingress is enabled, go to `https://artifact-metadata-repository.<ingress-domain>/play`.
```

Comment:
FYI, I had to use `http://` for me to access the playground. So maybe we need to include it could be http?